### PR TITLE
Docs

### PR DESCRIPTION
**Pull request by palango**
Update docs:
- Remove mention of broadcast room
- Remove mentions of refund transfers

### CODE REVIEW

File: docs/raiden_services.rst [hunk 1]
```diff
@@ -16,49 +17,78 @@ The Raiden network includes auxiliary services, that enhance the overall user ex
 Pathfinding Services
 ====================
 
-Pathfinding services have a global view on a token network can provide suitable payment paths for Raiden nodes.
+Pathfinding services have a global view on a token network can provide suitable
```
Comment (taleldayekh) [inline docs/raiden_services.rst]: **and** can provide suitable payment...

**Review by taleldayekh — APPROVED** [review]
Go for it 👍🏼

**Comment by palango** [comment]
> **and** can provide suitable payment... 

:eyes:

**Comment by palango** [comment]
Docs built, so merging...

**Pull request merged**